src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -121,370 +120,382 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
-                                                i18nKey="registration.files_and_license.version_accepted_helper_text"
-                                                components={[<Box component="span" sx={{ fontWeight: 'bold' }} />]}
-                                              />
-                                            </Typography>
-                                            <Typography paragraph>
-                                              <Trans
-                                                i18nKey="registration.files_and_license.version_published_helper_text"
-                                                components={[<Box component="span" sx={{ fontWeight: 'bold' }} />]}
-                                              />
-                                            </Typography>
-                                            <Typography paragraph>
-                                              <Trans
-                                                i18nKey="registration.files_and_license.version_publishing_agreement_helper_text"
-                                                components={[<Box component="span" sx={{ fontWeight: 'bold' }} />]}
-                                              />
-                                            </Typography>
-                                            <Typography paragraph>
-                                              <Trans
-                                                i18nKey="registration.files_and_license.version_embargo_helper_text"
-                                                components={[<Box component="span" sx={{ fontWeight: 'bold' }} />]}
-                                              />
-                                            </Typography>
-                                          </>
-                                        )}
-                                      </HelperTextModal>
-                                    </Box>
-                                  </TableCell>
-                                )}
-                                <TableCell>
-                                  <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center' }}>
-                                    {t('registration.files_and_license.license')}
-                                    <HelperTextModal
-                                      modalTitle={t('registration.files_and_license.licenses')}
-                                      modalDataTestId={dataTestId.registrationWizard.files.licenseModal}
-                                      buttonDataTestId={dataTestId.registrationWizard.files.licenseHelpButton}>
+              {files.length > 0 && (
+                <Box
+                  sx={{
+                    display: 'flex',
+                    flexDirection: 'column',
+                    gap: '1rem',
+                    mb: '2rem',
+                  }}>
+                  <TableContainer component={Paper} elevation={3}>
```

Comment:
Her er det en flex-box, men den har bare en `<TableContainer>` i seg, så da er flex overflødig (med mindre man bruker flex for å sentrere noe).
```suggestion
                <TableContainer component={Paper} elevation={3} sx={{ mb: '2rem' }}>
```
Ser at dette ikke er ditt verk, og var sånn før også, men greit å bare rette opp når vi først er her inne å fikler 😇

Reply:
Hva er vel bedre enn å be andre rette opp i mitt eget rot 😅